rules/manager.go

```diff
@@ -115,6 +117,22 @@ func NewGroupMetrics(reg prometheus.Registerer) *Metrics {
 			},
 			[]string{"rule_group"},
 		),
+		ingestionSamplesTotal: prometheus.NewCounterVec(
+			prometheus.CounterOpts{
+				Namespace: namespace,
+				Name:      "rule_result_ingestion_samples_total",
```

Comment:
rule_group_ingested_samples_total would be consistent with the existing metrics. The help should make clear that this is samples we attempted to ingest, as distinct from what was actually ingested.

Reply:
> Help:      "Total number of ruler result samples attempted to ingest."

I guess the ingested certainly implies that an ingestion has been successful (at least to me). I have used never the less in the last commit.

Would `s/prometheus_rule_group_ingested/rule_group_ingestion_samples_total` make it clearer?